packages/EasyTest/src/Trait/Messenger/MessengerAssertionsTrait.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyTest\Trait\Messenger;
```

Comment:
Do you think we need `Messenger` level right now? It seems excessive to me for now.

Reply:
Exactly, I agree. Refactored.